auditlog/vegetables-service/src/main/java/io/debezium/demos/auditing/vegetables/transactioncontext/TransactionContextData.java

```diff
@@ -0,0 +1,25 @@
+package io.debezium.demos.auditing.vegetables.transactioncontext;
+
+import java.time.ZonedDateTime;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="transaction_context_data")
+public class TransactionContextData {
+
+    @Id
+    @Column(name="transaction_id")
+    public long transactionId;
+
+    @Column(name="user_name")
+    public String userName;
+
+    @Column(name="client_date")
+    public ZonedDateTime clientDate;
+
+    public String usecase;
```

Comment:
nitpick -> shouldn't that also be `useCase` and then have an annotation for "use_case" as well to be consistent with naming?